### PR TITLE
Add task 1

### PR DESCRIPTION
**Pull request by katyachok**
task1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const express = require('express')
```
Comment (k-krasilnikova) [inline index.js]: Let's replace `require` with `import` es6

File: index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const app = express()
+const port = process.env.PORT || 5000
+
+app.get('/api/greetings/:name', (req, res) => {
+    res.status(200).send(`<h1>Hello ${req.params.name}</h1>`)
+})
+
+app.get('/api/greetings', (req, res) => {
+    res.status(400).json('please provide name (ex. /api/greeting/someName)')
+})
```
Comment (k-krasilnikova) [inline index.js]: Let's create an additional structure to the project:
```
src/main.js
src/modules
->src/modules/greetings
-->src/modules/greetings/constants/index.js
-->src/modules/greetings/routes/index.js
-->src/modules/greetings/controllers/sayHi.js
```
Both of the routes should be combined by common API prefix "/api/greetings" in main.js.
Routing of the greeting module should describe each route and execute right controller like middleware.
Also, could you please move all the messages/templates to the constants file?

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (katyachok) [reply index.js]: done

File: src/modules/greetings/controllers/sayHi.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { getHelloMessage, HELLO_MESSAGE_DESCRIPTION} from '../constants/index.js';
+
+export const sayHi = (req, res) => {
+	const { name } = req.params;
+	if (!name) res.status(400).json(HELLO_MESSAGE_DESCRIPTION);
+	else res.status(200).send(getHelloMessage(name));
```
Comment (k-krasilnikova) [inline src/modules/greetings/controllers/sayHi.js]: Could you please replace if with correct semantic view:
```
if (!name) {
 res.status(400).json(HELLO_MESSAGE_DESCRIPTION);
}
 res.status(200).send(getHelloMessage(name));
```

**Review by k-krasilnikova — APPROVED** [review]
Please one more change and PR is ready to be merged

**Pull request merged**